graphs/omen-thumbnailmapping/src/contract.ts

```diff
@@ -0,0 +1,15 @@
+import { ImageUpdated } from "../generated/Contract/Contract"
+import { OmenThumbnailMapping } from "./schema"
+
+export function handleImageUpdated(event: ImageUpdated): void {
+  let mapping = OmenThumbnailMapping.load(event.params.marketAddress)
+
+  if (!mapping) {
+    mapping = new OmenThumbnailMapping(event.params.marketAddress)
+  }
+
+  mapping.marketAddress = event.params.marketAddress
+  mapping.image_hash = event.params.image_hash
```

Comment:
I would expect `msg.sender` (present in event `ImageUpdated`) to also be stored.

Reply:
Is that available by default in the event? Or do you mean `changer` from `ImageUpdated`?

Reply:
Actually, `changer` and msg.sender is always the same: https://github.com/gnosis/labs-contracts/blob/8a0cd14919cfe09f287dfd495aae8f8b4db04968/src/OmenThumbnailMapping.sol#L22

So I will just add it 👌

Reply:
As you prefer, I think fetching `msg.sender` from event is cleaner but no strong preference

Reply:
Added 
![Screenshot by Dropbox Capture](https://github.com/gnosis/labs-contracts/assets/15619339/bc08785d-ba36-45de-8da3-02d56496d083)